reflex/components/core/upload.py

```diff
@@ -92,12 +92,33 @@ def cancel_upload(upload_id: str) -> EventSpec:
     return call_script(f"upload_controllers[{upload_id!r}]?.abort()")
 
 
+class UploadComponentUsed:
+    """A class to indicate whether an Upload component is used."""
+
+    is_used: ClassVar[bool] = False
```

Comment:
can this just be a ClassVar on the `UploadFilesProvider`?

Reply:
My first implementation was using a `ClassVar` but pyright did not like the generated pyi: https://github.com/reflex-dev/reflex/actions/runs/7603957212/job/20706298762

Reply:
oh gotcha; the pyi_generator script is broken 😢 

can you add `_used` or `_is_used` to the `EXCLUDED_PROPS` list around line 39 of the `pyi_generator.py` script. that will avoid putting the `ClassVar` into the signature of `.create` method. 

> `error: "ClassVar" is not allowed in this context`

This is basically saying you cannot annotate a function argument as a `ClassVar`, which makes perfect sense to me (but definitely was confusing when i first saw it).

Reply:
Yes i know, i even tried `EXCLUDED_PROPS` but then the ClassVar is excluded everywhere and hence pyright complains about the missing `_is_used` class attribute

Reply:
`  /home/bb/dev/reflex/github-bb-reflex/reflex/app.py:248:32 - error: Cannot access member "is_used" for type "type[UploadFilesProvider]"`

Reply:
@masenf i adjusted the pyi generator to exclude `ClassVars` and migrated `is_used` to `UploadFilesProvider`